src/qt/createwalletdialog.cpp

```diff
@@ -34,6 +34,15 @@ CreateWalletDialog::CreateWalletDialog(QWidget* parent) :
             ui->disable_privkeys_checkbox->setChecked(false);
         }
     });
+
+#ifndef USE_SQLITE
+    ui->descriptor_checkbox->setToolTip(tr("Compiled without sqlite support (required for descriptor wallets)"));
+    ui->descriptor_checkbox->setEnabled(false);
```

Comment:
What happened to setChecked(false)?

Reply:
Was based on yesterday's version of #20156. Rebased again.